spec/spec_helper.rb

```diff
@@ -88,7 +89,11 @@
       ex.run
     end
   end
-
+  
+  config.after do |example|
+    save_and_open_page if example.metadata[:type] == :feature and example.exception.present?
```

Comment:
>  Does that change your view @equivalentideas ?

Not really ;-)

> where you'd certainly want to see the page.

I'm always most focused on the test error, but maybe I should look at the page more. I'm happy to try this and we can always remove it if it gets in the way :+1: